PxCs/Interface/PxVm.cs

```diff
@@ -17,52 +17,112 @@ public enum PxVmInstanceType
             PxVmInstanceTypeNpc = 1,
             PxVmInstanceTypeItem = 2,
             PxVmInstanceTypeSfx = 3,
-            PxVmInstanceTypeMusic = 4
+            PxVmInstanceTypeMusic = 4,
+            PxVmInstanceTypeMenu = 5,
+            PxVmInstanceTypeMenuItem = 6
         };
 
         [Flags]
         public enum PxVmItemFlags
         {
             // Item categories
-            ITEM_KAT_NONE    = 1 <<  0, // misc
-            ITEM_KAT_NF      = 1 <<  1, // melee weapons
-            ITEM_KAT_FF      = 1 <<  2, // distant-combat weapons
-            ITEM_KAT_MUN     = 1 <<  3, // munition (->MultiSlot)
-            ITEM_KAT_ARMOR   = 1 <<  4, // armor and helmets
-            ITEM_KAT_FOOD    = 1 <<  5, // food (->MultiSlot)
-            ITEM_KAT_DOCS    = 1 <<  6, // documents
-            ITEM_KAT_POTIONS = 1 <<  7, // potions
-            ITEM_KAT_LIGHT   = 1 <<  8, // light sources
-            ITEM_KAT_RUNE    = 1 <<  9, // runes and scrolls
-            ITEM_KAT_MAGIC   = 1 << 31, // rings and amulets
-            ITEM_KAT_KEYS    = ITEM_KAT_NONE,
-            
+            ITEM_KAT_NONE = 1 << 0, // misc
+            ITEM_KAT_NF = 1 << 1, // melee weapons
+            ITEM_KAT_FF = 1 << 2, // distant-combat weapons
+            ITEM_KAT_MUN = 1 << 3, // munition (->MultiSlot)
+            ITEM_KAT_ARMOR = 1 << 4, // armor and helmets
+            ITEM_KAT_FOOD = 1 << 5, // food (->MultiSlot)
+            ITEM_KAT_DOCS = 1 << 6, // documents
+            ITEM_KAT_POTIONS = 1 << 7, // potions
+            ITEM_KAT_LIGHT = 1 << 8, // light sources
+            ITEM_KAT_RUNE = 1 << 9, // runes and scrolls
+            ITEM_KAT_MAGIC = 1 << 31, // rings and amulets
+            ITEM_KAT_KEYS = ITEM_KAT_NONE,
+
```

Comment:
Please stop your formatter from reformatting all the lines. ;-)

e.g. these lines were intentionally intended this way.